### PR TITLE
runtime: clh-config: add runtime hooks to the clh toml

### PR DESCRIPTION
**Pull request by egernst**
Today hooks are only described in the QEMU toml. This shouldn't be VMM
specific -- let's make sure these are advertised for Cloud Hypervisor as
well.

Fixes: #1401

Signed-off-by: Eric Ernst <eric.g.ernst@gmail.com>

### CODE REVIEW

**Review by mxpv — APPROVED** [review]

**Comment by egernst** [comment]
Turns out we weren't setting it in the hypervisor config either for CLH. Verified with these changes we now see hooks executed when running with CLH. Will add my local test details to https://github.com/kata-containers/tests/issues/2632 so we can get this covered for at least QEMU/CLH.

**Comment by egernst** [comment]
basic test to verify: https://github.com/kata-containers/tests/issues/2632#issuecomment-777865136

File: src/runtime/cli/config/configuration-qemu.toml.in [hunk 1]
```diff
@@ -309,7 +309,7 @@ pflashes = []
 # lexicographical order, to the lifecycle of the guest container.
 # Hooks are executed in the runtime namespace of the guest. See the official documentation:
 # https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#posix-platform-hooks
-# Warnings will be logged if any error is encountered will scanning for hooks,
+# Warnings will be logged if any error is encountered while scanning for hooks,
```
Comment (fidencio) [inline src/runtime/cli/config/configuration-qemu.toml.in:312]: Eric, could you also do the same fix for the https://github.com/kata-containers/kata-containers/blob/14bb24e4ca4d097c6374901760a50183026682ab/src/runtime/cli/config/configuration-acrn.toml.in#L116 ?

File: src/runtime/pkg/katautils/config.go [hunk 2]
```diff
@@ -873,6 +873,7 @@ func newClhHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		HotplugVFIOOnRootBus:    h.HotplugVFIOOnRootBus,
 		PCIeRootPort:            h.PCIeRootPort,
 		DisableVhostNet:         true,
+		GuestHookPath:           h.guestHookPath(),
```
Comment (fidencio) [inline src/runtime/pkg/katautils/config.go:876]: Ouch, this one deserves its own patch.
Nice catch!

**Review by fidencio — APPROVED** [review]
@egernst, a few comments, those are mostly nitpicks, feel free to address them if you have cycles to do so.
